tests/components/dialogflow/test_init.py

```diff
@@ -16,19 +16,25 @@
 REQUEST_TIMESTAMP = "2017-01-21T17:54:18.952Z"
 CONTEXT_NAME = "78a5db95-b7d6-4d50-9c9b-2fc73a5e34c3_id_dialog_context"
 
-calls = []
-
 
 @pytest.fixture
-async def fixture(hass, aiohttp_client):
-    """Initialize a Home Assistant server for testing this module."""
+async def calls(hass, fixture):
```

Comment:
I didn't think that coroutines were supported as pytest fixtures without using pytest-asyncio.

Reply:
That is a good point, yet it passes…